test/e2e/util.go

```diff
@@ -312,18 +312,19 @@ func logf(level string, format string, args ...interface{}) {
 }
 
 func Logf(format string, args ...interface{}) {
-	logf("INFO", format, args...)
+	msg := fmt.Sprintf(format, args...)
+	logf("INFO %v", msg)
```

Comment:
Line 310 (just above here) defines logf.  Do you want me to create a similar function called log that does an FPrint to GingkoWriter?

Or call logf and pass an extra "%v" as a param to it?